docs/reference/07-delete.md

```diff
@@ -0,0 +1,41 @@
+# Create records
+
+Find, update and delete records.
```

Comment:
Fix this line? I think this guide only talks about deleting. Same with the above.